skimage/draw/_random_shapes.py

```diff
@@ -103,11 +103,17 @@ def _generate_circle_mask(point, image, shape, random):
     if available_radius < min_radius:
         raise ArithmeticError('cannot fit shape to image')
     radius = random.randint(min_radius, available_radius + 1)
-    circle = draw_circle(point[0], point[1], radius)
+    # TODO: think about how to deprecate this
+    # Unfortunately, renaming `circle` to `disk` directly would break
+    # downstream code that switches between `circle` and disk.
```

Comment:
This comment isn't super clear. I think it's easier to say just "switching to 'disk' here would be a breaking change for downstream libraries".

I'm also not positive about this PR in general... "square" and "triangle" are technically not filled either...

Reply:
I can restrict this PR to the internal warnings, as opposed to an API change. I would prefer that anyway ;)